tests/e2e_v2/ui/README.md

```diff
@@ -0,0 +1,225 @@
+# RHOAI Dashboard UI Tests
+
+This directory contains UI tests for the RHOAI Dashboard, specifically targeting the Distributed Workloads page to verify Ray cluster visibility and functionality.
+
+## Overview
+
+The UI tests use Selenium WebDriver with Chrome (headless) to automate browser interactions with the RHOAI Dashboard. They are designed to work in conjunction with the upgrade tests to verify that Ray clusters created before an upgrade remain visible and functional after the upgrade.
+
+## Test Structure
+
+```
+tests/ui/
+├── conftest.py                              # Pytest fixtures for Selenium setup
+├── pages/
+│   └── distributed_workloads_page.py        # Page Object Model for Distributed Workloads page
+└── README.md                                # This file
+
+tests/upgrade/
+├── 01_raycluster_sdk_upgrade_test.py        # Pre/post upgrade backend tests (runs first)
+├── 02_dashboard_ui_upgrade_test.py          # Pre/post upgrade UI tests (runs second)
+└── conftest.py                              # Imports UI fixtures for upgrade tests
+```
+
+**Note**: Test files are prefixed with numbers (`01_`, `02_`) to ensure proper execution order:
+1. First, the Ray cluster is created (`01_raycluster_sdk_upgrade_test.py`)
+2. Then, the UI tests verify the cluster appears in the dashboard (`02_dashboard_ui_upgrade_test.py`)
+
+## Prerequisites
+
+### Python Dependencies
+
+The UI tests require the following dependencies (already added to `pyproject.toml`):
+
+- `selenium >= 4.27.1` - Browser automation framework
```

Comment:
Is the use of Selenium new here or was it being used in other tests before new structure?

Reply:
@laurafitzgerald actually all of the UI and upgrade tests added in this PR were just copied from the existing ones. They can pretty much be ignored for this PR I think